README.md

```diff
@@ -458,31 +455,35 @@ Now import the owner private key generated previously in the wallet:
 ```
 
 <a name="uploadsmartcontract"></a>
-### Upload sample contract to blockchain 
+### Upload sample "currency" contract to blockchain 
 
-Before uploading a contract, you can verify that there is no current contract:
+Before uploading a contract, we will verify that there is no current contract:
```

Comment:
Why did we change to 'we' here? It was 'you' before and based on kubernetes (we were adopting these) style-guides we should be using 'you' where applicable.

Reply:
Engaging language vs passive language. This is a walkthrough which is specialized case.